src/fe-small-advertisements/src/actions/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './name';
```

Comment:
`name` is such a generic name. Can we use something more specific in this case?

Reply:
true. Even the linter complains sometimes when one uses `name` in some places.